src/components/AppNavigation/AppointmentConfigList/AppointmentConfigListItem.vue

```diff
@@ -99,16 +101,18 @@ export default {
 			loading: false,
 		}
 	},
+	computed: {
+		hasClipboard() {
+			return navigator && navigator.clipboard
+		},
+	},
 	methods: {
 		closeModal() {
 			this.showModal = false
 		},
 		copyLink() {
-			if (!navigator || !navigator.clipboard) {
-				return
-			}
-
 			navigator.clipboard.writeText(this.config.bookingUrl)
```

Comment:
this returns a promise. could you await it and wrap it in a try-catch? ref https://github.com/nextcloud/calendar/issues/3739#issuecomment-981516884 dunno if @miaulalala worked on this already

Reply:
on error please also show a toast

Reply:
TIL! Will add an await and some error output.